runtime/src/accounts_db.rs

```diff
@@ -2466,12 +2466,19 @@ impl AccountsDb {
     }
 
     pub fn shrink_all_slots(&self, is_startup: bool) {
+        const MAX_DIRTY_STORES: usize = 10_000;
```

Comment:
nit: naming: `INTERLEAVED_CLEANING_THRESHOLD`?

Reply:
renamed to `DIRTY_STORES_CLEANING_THRESHOLD`